requirements/dev.in

```diff
@@ -1,3 +1,3 @@
--e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos[dev]
+-e git+https://github.com/kytos-ng/python-openflow.git@upgrade/python#egg=python-openflow
```

Comment:
Reminder to revert this pinned branch before merging

Reply:
Reverted